sass/config/_fonts.scss

```diff
@@ -15,13 +15,14 @@ $font-formats: 'woff2' 'woff' 'ttf';
 /// - suppy additional html head as a nested block
 ///
 /// @group fonts
-/// @font 'typekit' (regular, italic, bold, bold italic)
+/// @font typekit (regular, italic, bold, bold italic)
 ///   <script src="https://use.typekit.net/slx1xnq.js"></script>
 ///   <script>try{Typekit.load({ async: true });}catch(e){}</script>
 $typekit-font: (
   'name': 'freight-text-pro',
   'stack': 'fantasy',
   'source': 'https://typekit.com/fonts/freight-text',
+  'local': 'Freight Text Pro' 'freight-text-pro',
```

Comment:
@mirisuzanne Am I correct that we want `local` added all the time? If this is defined, we use it, but if it's missing we still include it just with the `name`?

Reply:
Had to do a bit of research, mainly here:

https://www.paulirish.com/2009/bulletproof-font-face-implementation-syntax/

and then looking at the FontSpring approach. This seems to be their most recent:

http://blog.fontspring.com/2011/02/further-hardening-of-the-bulletproof-syntax/

To summarize the relevant part, some people don't trust `local()` because it *may* load something slightly different than the font you are delivering. Seems like we should suppot turning it on or off like any other format.

Reply:
Ah, so we only add it if truthy. Got it.

Reply:
Yeah, though I'm currently allowing this to be set globally (add `local` to `$font-formats`) or local to the font.

Reply:
How does that work for Herman? As far as I'm aware, Herman currently ignores the global `$font-formats` entirely, no?

Reply:
oh, huh. Is it just checking a folder for all files that match a pattern? Maybe I need to rethink this. Sorry I can't really dig in farther before my talk this morning.

Reply:
My instinct is that we don't gain much from the global `$font-format: local`, and can just go by the truthy `local` font attribute. I don't know if that impacts the other change in this branch – allowing each format url to be described individually.

Reply:
Right now for local fonts, Herman requires that you pass in the formats when you call `@font`, e.g. `/// @font local {woff2, woff, ttf}`. The `@font-formats` global is only used by Accoutrement, as far as I know.

I'm not sure I understand what "allowing each format url to be described individually", or how it relates to this.

(No rush to respond.)